README.md

```diff
@@ -71,7 +71,7 @@ Install [geckodriver](https://github.com/mozilla/geckodriver/releases) and
 [chromedriver](https://sites.google.com/a/chromium.org/chromedriver/downloads) somewhere
 on your `PATH`.
 
-`pytest test/`
+`pytest test/ packages/`
```

Comment:
I wonder if we should recommend `make test` instead here...

Reply:
> I wonder if we should recommend make test instead here...

I think using pytest directly is better because then one can select specific tests with `-k` flag etc.